src/analysis/inner_product_space/orientation.lean

```diff
@@ -37,7 +37,7 @@ protected def orientation.fin_orthonormal_basis {n : ℕ} (hn : 0 < n) (h : finr
 begin
```

Comment:
Hi Heather,
Doing this actually creates _a lot_ of errors, including ones that would include a refactor of lemmas such as:
 - `basis.map_isometry_euclidean_of_orthonormal` should be a result about orthonormal bases
 - `complex.map_isometry_of_orthonormal` should be a result along the lines:  `complex.map_isometry_of_orthonormal_basis`
 - etc.
Because the definition of `orthonormal.oangle` should be one with `orthonormal_basis.oangle` _instead_ so that this `def` as changed applies correctly. (Instead having a bazillion examples of `orthonormal_basis.orthonormal.oangle`, which actually also doesn't work because it wants a term of type `orthonormal ℝ (orthonormal_basis.to_basis)`)
However, I suspect this will involve creating a lot more API (like `orthonormal_basis.map`), which I won't have time to do for at least a month...

I would like to get this commit in _now_ so that people (e.g. Anatole) can use it, and so that I don't need to return to this in a month and spend a few days just getting it up to date with the changes in the mean time.

There might be a "hacky" way to get this in by just casting everything to basis carefully, but that I'm worried that the cure there might be worse than the symptoms here. This is my take--I'm happy to be convinced otherwise, though!

Reply:
OK, let's save this for later then!  You can revert to the last working state and then merge.

Reply:
Great! Sounds good. Thanks for understanding.